README.md

```diff
@@ -184,7 +185,8 @@ bbgo sync --session binance
 
 Check out the strategy directory [strategy](pkg/strategy) for all built-in strategies:
 
-- `pricealert` strategy demonstrates how to use the notification system [pricealert](pkg/strategy/pricealert)
+- `pricealert` strategy demonstrates how to use the notification system [pricealert](pkg/strategy/pricealert). See 
+  [Price Alert Strategy](./doc/strategy/pricealert.md).
```

Comment:
rename label `[Price Alert Strategy]` to just `[doc]`? because when the markdown is rendered, there will be two "Price Alert" links and it might be confusing

Reply:
submitted a new commit to solve this